DEVDOC.md

```diff
@@ -116,6 +116,42 @@ lookups.
 
 Any help to fix this is very welcome!
 
+#### Integration with VSCode and Delve
+
+If you would like to use the delve debugger for Go with VSCode to debug `gapis`, You can follow these steps
+
+1. Complete the fusing steps.
+
+2. Create a launch.json under the project directory `Ctrl + Shift + P` and `Debug: Open launch.json`
+
+3. Paste this as one of the configuration. This will ensure that you have a setup for attaching to Delve when you launch the project.
+```
+{
+      "name": "Attach to Delve",
+      "type": "go",
+      "request": "attach",
+      "mode": "remote",
+      "apiVersion": 2,
+      "remotePath": "gapis/",
+      "cwd": "${workspaceFolder}/src/github.com/google/gapid/gapis",
+      "dlvLoadConfig": {
+          "followPointers": true,
+          "maxVariableRecurse": 1,
+          "maxStringLen": 120,
+          "maxArrayValues": 120,
+          "maxStructFields": -1
+      },
+      "host": "127.0.0.1",
+      "port": 1234,
+    },
+```
+
+4. Start delve in headless mode at your gapid folder. `dlv exec --headless --listen=127.0.0.1:1234 --api-version 2 ./bazel-bin/pkg/gapis` This will allow you to use port 1234 to connect to delve from VSCode.
```

Comment:
"at your gapid folder" is ambiguous, let's be precise now that we have two folders (the git checkout and the gofuse one). Also, prefer to put command line examples on their own line, it makes it easier to copy-paste them. Finally, let's reuse the same flags as the example in the previous section. A possible rephrasing:

Start delve in headless mode, making sure to use the same port as the one in the configuration above, e.g. from the gapid checkout directory:
```
dlv exec --headless --listen=127.0.0.1:1234 --api-version 2 ./bazel-bin/pkg/gapis -- -enable-local-files -persist -rpc localhost:8888
```

Reply:
I intentionally have not put the gapis parameters for two reasons;

1. They are irrelevant. 
2. They may change and then we would need to update it. 

It should be easy to figure out how to add it by reading the other parts of the doc.